src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace TEC\Tickets\Commerce;
+
+use TEC\Tickets\Settings as Tickets_Settings;
+use \tad_DI52_ServiceProvider;
+
+/**
+ * Class Payments_Tab
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce
+ */
+class Payments_Tab extends tad_DI52_ServiceProvider {
+
+	/**
+	 * @inheritdoc
+	 */
+	public function register() {
+		$this->register_hooks();
+	}
+
+	/**
+	 * Registers the all the 1st level filters and actions for Payments Tab.
+	 *
+	 * @since TBD
+	 */
+	public function register_hooks() {
+		add_action( 'tribe_settings_do_tabs', [ $this, 'register_tab' ], 15 );
+	}
```

Comment:
Move this hooks here to the `Hooks.php` file from the `\TEC\Tickets` level, calling directly the register tab.